MarsQACompetitionTaskNUnit/Tests/CertificationTest.cs

```diff
@@ -0,0 +1,144 @@
+﻿using MarsQACompetitionTaskNUnit.Assertions;
+using NUnit.Framework;
+
+namespace MarsQACompetitionTaskNUnit.Tests
+{
+    [Parallelizable]
+    [TestFixture]
+    public class CertificationTest: BaseTest 
+    {
+        [SetUp]
+        public void SetUpCertification()
+        {           
+            certificationPageObject.NavigateToCertificationTab();
+            Thread.Sleep(1000);
+        }
+
+        [Test, Order(1), Description("This test create a new Certification record")]
+        public void TestCreateCertificationRecord()
+        {         
+            certificationPageObject.ClearCertification();
+            certificationPageObject.CreateCertificationRecord(certificationConfig[0].Certificate, certificationConfig[0].From, certificationConfig[0].Year);
+            AssertionHelpers.AssertToolTipMessage(certificationPageObject, certificationConfig[0].AssertionMessage);
+            bool recordPresent = certificationPageObject.IsCertificationRecordPresent(certificationConfig[0].Certificate, certificationConfig[0].From, certificationConfig[0].Year);
+            Assert.IsTrue(recordPresent);
+            certificationPageObject.ClearCertification();
+        }
+
+        [Test, Order(2), Description("This test edit  Certification record")]
+        public void TestEditCertificationRecord()
+        {
+            certificationPageObject.ClearCertification();
+            certificationPageObject.CreateCertificationRecord(certificationConfig[0].Certificate, certificationConfig[0].From, certificationConfig[0].Year);
+            Thread.Sleep(1000);
+            certificationPageObject.EditCertificationRecord(certificationConfig[6].Certificate, certificationConfig[6].From, certificationConfig[6].Year);
```

Comment:
I would not pass (string Certificate, string From, string Year) instead pass the certification and use what you need imagine if tomorrow you need something else then you can update the pojo instead of the method signature

Reply:
Updated the Pojo in the Pages.